apps/webpack.js

```diff
@@ -1,7 +1,12 @@
 var _ = require('lodash');
 var webpack = require('webpack');
 var path = require('path');
-var config = {
+var LiveReloadPlugin = require('webpack-livereload-plugin');
+
+var AUTO_RELOAD = ['true', '1'].indexOf(process.env.AUTO_RELOAD) !== -1;
```

Comment:
I think I like the idea of this latter approach...makes it so that I can go to one place to find everything I can potentially set.